proto/c1/connector/v2/annotation_entitlement.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package c1.connector.v2;
+
+import "google/protobuf/struct.proto";
+
+option go_package = "github.com/conductorone/baton-sdk/pb/c1/connector/v2";
+
+message EntitlementImmutable {
+  optional string source_id = 1;
```

Comment:
minor: i wasn't sure we want to use the optional flag in proto3 -- i think tooling adoption is messy?

Reply:
Fixed in 2c96485ca651ed96d835ceb70ddf53c17d2955da